src/status_im/contexts/wallet/account/view.cljs

```diff
@@ -50,7 +51,9 @@
              :receive-action #(rf/dispatch [:open-modal :wallet-share-address {:status :receive}])
              :buy-action     #(rf/dispatch [:show-bottom-sheet
                                             {:content buy-drawer}])
-             :bridge-action  #(rf/dispatch [:open-modal :wallet-bridge])}])
+             :bridge-action  (if (:bridge-token config/wallet-feature-flags)
```

Comment:
To improve readability, maybe would be nice to have some function like `feature-enabled?` that receives the feature key, located in a utils file. WDYT?

Reply:
yeah, I agree! There was discussion in another pr - https://github.com/status-im/status-mobile/pull/18530#discussion_r1457285073
where we will put some effort into how we want to work with feature flags. 
For now I will leave it this way but in an upcoming pr/issue will refactor this

Reply:
Awesome 😃